docs/docker-compose-mysql.md

```diff
@@ -29,6 +29,8 @@ git clone https://github.com/owid/owid-content
 
 ## Starting our development environment
 
+On **MacOS** make sure you're running the Docker Software, which is not on the command line like everything else. This error may appear `Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?` Your 0 TMux is probably broken too.
```

Comment:
I think just remove TMux, the error is definitely that Docker wasn't running, so TMux seems confusing to me now - I don't have evidence TMux wasn't interacting with Docker beyond the fact it can't interact if it's not even there.

Reply:
I felt I should be a bit more specific "Docker is not running" has a good chance to keeping me in that terminal frame of mind.

"MacOS applications folder Docker software" is clearer IMO. But I'll not continuing pushing it on a first PR.

Reply:
I assume this requires a review from a codebase owner or somesuch?